packages/config/src/Config.ts

```diff
@@ -501,6 +501,14 @@ export function readConfigJson(
     exp.version = pkg.version;
   }
 
+  if (exp && !exp.sdkVersion) {
+    try {
+      const packageJsonPath = resolveModule('expo/package.json', projectRoot, exp);
+      const expoPackageJson = require(packageJsonPath);
+      exp.sdkVersion = expoPackageJson.version;
```

Comment:
need to zero out the minor and patch versions. could cause issues if people have sdkVersion: 35.0.1 unintentionally. so for now i think zeroing out the version is safest.